src/dune_project.ml

```diff
@@ -215,6 +215,7 @@ module Extension = struct
     }
 
   let extensions = Hashtbl.create 32
+  let instanciated_extensions = Hashtbl.create 32
```

Comment:
This should be per-project, otherwise it won't work when there are several projects in the same workspace.

Reply:
It's now stored in the project.